.gitattributes

```diff
@@ -1,3 +1,4 @@
+* text=auto eol=lf
```

Comment:
Do we really want to do this for all files in the repo? PNG files are not text. SVG files are XML files which are text, but that is not the most important type to give them.